### PR TITLE
📝 Add plot to show downloads datasets

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

This PR add plots to show the downloads datasets.

 

### Details

image

### CODE REVIEW

File: pyvista/examples/downloads.py [hunk 1]
```diff
@@ -2901,7 +3216,10 @@ def download_mars_jpg():  # pragma: no cover
     Examples
     --------
     >>> from pyvista import examples
-    >>> dataset = examples.download_mars_jpg()  # doctest:+SKIP
+    >>> dataset = examples.download_mars_jpg()
+    >>> dataset.plot(cpos="iso")
+    >>> examples.delete_downloads()
+    True
```
Comment (tkoyama010) [inline pyvista/examples/downloads.py:3222]:
```suggestion
    >>> import pyvsita as pv
    >>> pl = pv.Plotter()
    >>> dataset = examples.download_mars_jpg()
    >>> dataset.plot(cpos="iso")
    >>> examples.delete_downloads()
    True
```

File: pyvista/examples/downloads.py [hunk 2]
```diff
@@ -2617,7 +2899,10 @@ def download_sky_box_cube_map():  # pragma: no cover
     Examples
     --------
     >>> from pyvista import examples
-    >>> dataset = examples.download_sky_box_cube_map()  # doctest:+SKIP
+    >>> dataset = examples.download_sky_box_cube_map()
+    >>> dataset.plot(cpos="iso")
+    >>> examples.delete_downloads()
+    True
```
Comment (tkoyama010) [inline pyvista/examples/downloads.py:2905]:
```suggestion
    >>> import pyvsita as pv
    >>> pl = pv.Plotter()
    >>> dataset = examples.download_sky_box_cube_map()
    >>> dataset.plot(cpos="iso")
    >>> examples.delete_downloads()
    True
```

File: pyvista/examples/downloads.py [hunk 3]
```diff
@@ -2918,7 +3236,10 @@ def download_stars_jpg():  # pragma: no cover
     Examples
     --------
     >>> from pyvista import examples
-    >>> dataset = examples.download_stars_jpg()  # doctest:+SKIP
+    >>> dataset = examples.download_stars_jpg()
+    >>> dataset.plot(cpos="iso")
+    >>> examples.delete_downloads()
+    True
```
Comment (tkoyama010) [inline pyvista/examples/downloads.py:3242]:
```suggestion
    >>> import pyvsita as pv
    >>> pl = pv.Plotter()
    >>> dataset = examples.download_stars_jpg()
    >>> dataset.plot(cpos="iso")
    >>> examples.delete_downloads()
    True
```

File: pyvista/examples/downloads.py [hunk 4]
```diff
@@ -2617,7 +2899,12 @@ def download_sky_box_cube_map():  # pragma: no cover
     Examples
     --------
     >>> from pyvista import examples
-    >>> dataset = examples.download_sky_box_cube_map()  # doctest:+SKIP
+    >>> import pyvsita as pv
```
Comment (tkoyama010) [inline pyvista/examples/downloads.py:2902]:
```suggestion
    >>> import pyvista as pv
```

File: pyvista/examples/downloads.py [hunk 5]
```diff
@@ -2901,7 +3218,12 @@ def download_mars_jpg():  # pragma: no cover
     Examples
     --------
     >>> from pyvista import examples
-    >>> dataset = examples.download_mars_jpg()  # doctest:+SKIP
+    >>> import pyvsita as pv
```
Comment (tkoyama010) [inline pyvista/examples/downloads.py:3221]:
```suggestion
    >>> import pyvista as pv
```

File: pyvista/examples/downloads.py [hunk 6]
```diff
@@ -2918,7 +3240,12 @@ def download_stars_jpg():  # pragma: no cover
     Examples
     --------
     >>> from pyvista import examples
-    >>> dataset = examples.download_stars_jpg()  # doctest:+SKIP
+    >>> import pyvsita as pv
```
Comment (tkoyama010) [inline pyvista/examples/downloads.py:3243]:
```suggestion
    >>> import pyvista as pv
```

File: requirements_docs.txt [hunk 7]
```diff
@@ -30,3 +30,4 @@ trimesh
 typed-ast
 typing-extensions
 pythreejs
+osmnx
```
Comment (tkoyama010) [inline requirements_docs.txt:33]:
```suggestion
```

File: pyvista/examples/downloads.py [hunk 8]
```diff
@@ -2675,7 +2965,10 @@ def download_gpr_data_array(load=True):  # pragma: no cover
     Examples
     --------
     >>> from pyvista import examples
-    >>> dataset = examples.download_gpr_data_array()  # doctest:+SKIP
+    >>> dataset = examples.download_gpr_data_array()
+    >>> dataset.plot(cpos="iso")
```
Comment (tkoyama010) [inline pyvista/examples/downloads.py:2969]:
```suggestion
    >>> import matplotlib.pyplot as plt
    >>> dataset = examples.download_gpr_data_array()
    >>> plt.figure(figsize=(15,3))
    >>> plt.pcolormesh(dataset, cmap="seismic", clim=[-1,1])
    >>> plt.gca().invert_yaxis()
```

File: requirements_test.txt [hunk 9]
```diff
@@ -19,3 +19,4 @@ param==1.10.1
 ipygany
 pytest-xdist
 pythreejs
+osmnx==1.1.1
```
Comment (tkoyama010) [inline requirements_test.txt:22]:
```suggestion
osmnx
```

File: pyvista/examples/downloads.py [hunk 10]
```diff
@@ -2901,7 +3221,12 @@ def download_mars_jpg():  # pragma: no cover
     Examples
     --------
     >>> from pyvista import examples
-    >>> dataset = examples.download_mars_jpg()  # doctest:+SKIP
+    >>> import pyvista as pv
+    >>> pl = pv.Plotter()
+    >>> dataset = examples.download_mars_jpg()
+    >>> dataset.plot(cpos="iso")
```
Comment (tkoyama010) [inline pyvista/examples/downloads.py:3227]:
```suggestion
    >>> import pyvista as pv
    >>> pl = pv.Plotter()
    >>> dataset = examples.download_mars_jpg()
    >>> pl.add_background_image(dataset)
    >>> pl.show()
```

File: requirements_test.txt [hunk 11]
```diff
@@ -19,3 +19,4 @@ param==1.10.1
 ipygany
 pytest-xdist
 pythreejs
+osmnx
```
Comment (tkoyama010) [inline requirements_test.txt:22]:
```suggestion
```

File: pyvista/examples/downloads.py [hunk 12]
```diff
@@ -2617,7 +2899,13 @@ def download_sky_box_cube_map():  # pragma: no cover
     Examples
     --------
     >>> from pyvista import examples
-    >>> dataset = examples.download_sky_box_cube_map()  # doctest:+SKIP
+    >>> import pyvista as pv
+    >>> pl = pv.Plotter()
+    >>> dataset = examples.download_sky_box_cube_map()
+    >>> pl.add_background_image(dataset)
```
Comment (tkoyama010) [inline pyvista/examples/downloads.py:2905]:
```suggestion
    >>> pl.add_actor(dataset.to_skybox())
    >>> _ = pl.add_actor(dataset.to_skybox())
    >>> pl.set_environment_texture(dataset)
    >>> pl.show()
```

File: pyvista/examples/downloads.py [hunk 13]
```diff
@@ -2617,7 +2899,16 @@ def download_sky_box_cube_map():  # pragma: no cover
     Examples
     --------
     >>> from pyvista import examples
-    >>> dataset = examples.download_sky_box_cube_map()  # doctest:+SKIP
+    >>> import pyvista as pv
+    >>> pl = pv.Plotter()
+    >>> dataset = examples.download_sky_box_cube_map()
+    >>> pl.add_actor(dataset.to_skybox())
```
Comment (tkoyama010) [inline pyvista/examples/downloads.py:2905]:
```suggestion
```

File: pyvista/examples/downloads.py [hunk 14]
```diff
@@ -2617,7 +2899,15 @@ def download_sky_box_cube_map():  # pragma: no cover
...
     --------
     >>> from pyvista import examples
-    >>> dataset = examples.download_sky_box_cube_map()  # doctest:+SKIP
+    >>> import pyvista as pv
+    >>> pl = pv.Plotter()
+    >>> dataset = examples.download_sky_box_cube_map()
+    >>> _ = pl.add_actor(dataset.to_skybox())
+    >>> pl.set_environment_texture(dataset)
+    >>> pl.show()
+    >>> pl.show()
```
Comment (tkoyama010) [inline pyvista/examples/downloads.py:2908]:
```suggestion
```

**Comment by MatthewFlamm** [comment]
It is now messy either way, but it might be nice to completely remove the unintended commits.  Otherwise the other author will be involved in this PR.

**Comment by tkoyama010** [comment]
Thank you for your comment. I close this PR and open #1826 .

**Pull request closed without merging**